Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -1231,6 +1311,30 @@ public void When_excluding_virtual_or_abstract_property_exclusion_works_properly
                 .Excluding(o => o.VirtualProperty)
                 .Excluding(o => o.DerivedProperty2));
         }
+
+        [Fact]
+        public void Exluding_fields_ignores_them()
+        {
+            object class1 = new ClassWithSomeFieldsAndProperties
+            {
+                Field1 = "Lorem",
+                Field2 = "ipsum",
+                Field3 = "dolor",
+                Property1 = "sit",
+                Property2 = "amet",
+                Property3 = "consectetur"
+            };
+
+            object class2 = new ClassWithSomeFieldsAndProperties
+            {
+                Property1 = "sit",
+                Property2 = "amet",
+                Property3 = "consectetur"
+            };
+
+            // Act / Assert
+            class1.Should().BeEquivalentTo(class2, opts => opts.ExcludingFields().RespectingRuntimeTypes());
```

Comment:
🔧 Even if you would not exclude the fields, this test would still succeed. 
🙃 We prefer to use terms like `subject` and `expectation`

Reply:
I don't get it. If I remove the `ExcludeProperties()` the test fails.
Am I missing something?

Reply:
Hmm, I was convinced fields were excluded by default, but it turns out I was wrong. Sorry about that.

Reply:
No problem :)